README.md

```diff
@@ -2,53 +2,66 @@
 Unofficial Ruby client for the Megaphone API
 
 ## Installation
-    gem 'megaphone_client', github:"scpr/megaphone_client"
+```bash
+gem 'megaphone_client', github:"scpr/megaphone_client"
+```
 
 ## Usage
+**Note:** Megaphone API props, such as `externalId`, is in camelCase instead of snake_case because Megaphone's API expects it when accessing their API. So when passing params or putting/posting a hash, use camelCase. When interfacing with the gem's API, use snake_case.
+
 ### Configuration
 Configure your app to connect to Megaphone, either in an initializer or your environment files:
 
 ```ruby
-  MegaphoneClient.setup do |config|
-      config.token = "{megaphone api token}"
-      config.network_id = "{megaphone network id}"
-      config.organization_id = "{megaphone organization id}"
-  end
+@megaphone = MegaphoneClient.new({
+  token: "{megaphone api token}",
+  network_id: "{megaphone network id}",
+  organization_id: "{megaphone organization id}"
+})
 ```
 
 ### Searching
 
+**Note:** the property `externalId` is written in camelCase because it's expected by the Megaphone API
+
 ```ruby
-Megaphone::Episode.search({ externalId: obj_key })
+@megaphone.episode.search({ externalId: obj_key })
```

Comment:
Curious, but what do you think about calling it like `@megaphone.episodes.search()`, since it's searching the episodes collection, and not searching in a specific episode?